sources/lib/DependencyInjection/PommExtension.php

```diff
@@ -39,9 +39,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services/pomm.yml');
 
-        if ($container->hasParameter('data_collector.time.class')) {
-            $loader->load('services/profiler.yml');
-        }
+        $loader->load('services/profiler.yml');
```

Comment:
The problem was that the .class parameter was used. In sf v3, all *.class parameters are gone, and the class name is written directly into the configuration.